sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/testing/TestPipelineArgs.java

```diff
@@ -0,0 +1,114 @@
+/*
```

Comment:
Don't like the name `TestPipelineArgs`. We have `TestPipeline` concept already, but this is unrelated to that.